lib/shipit.rb

```diff
@@ -85,6 +85,10 @@ def enable_samesite_middleware?
     ENV['SHIPIT_ENABLE_SAMESITE_NONE'].present?
   end
 
+  def obey_bare_shipit_file?
+    ActiveModel::Type::Boolean.new.cast(ENV['SHIPIT_OBEY_BARE_SHIPIT_FILE'] || true)
```

Comment:
_Really_ small nitpick, but the repetition of `SHIPIT` looks a bit weird to me. What do you think about calling it `SHIPIT_OBEY_BARE_CONFIG_FILE`?